azurecompute/src/main/java/org/jclouds/azurecompute/domain/DeploymentParams.java

```diff
@@ -113,6 +113,17 @@ public static ExternalEndpoint inboundUdpToLocalPort(final int port, final int l
    @Nullable
    public abstract String virtualNetworkName();
 
+   /**
+    * Optional. Specifies the name of a reserved IP address that is to be assigned to the deployment. You must run
+    * Create Reserved IP Address before you can assign the address to the deployment using this element.
+    *
+    * The ReservedIPName element is only available using version 2014-05-01 or higher.
+    *
+    * @return reserved IP.
+    */
+   @Nullable
+   public abstract String reservedIPName();
```

Comment:
It could be also convenient to annotate this with [SinceApiVersion](https://github.com/jclouds/jclouds/blob/master/core/src/main/java/org/jclouds/rest/annotations/SinceApiVersion.java).